frontend/src/hooks/useStore.ts

```diff
@@ -48,7 +49,14 @@ export const useStore = create<State>((set, get) => ({
     updatePersonalStatus: (workshopId: string, personalStatus: PersonalStatus) => {
         axios.put(`/api/workshop/${workshopId}`, personalStatus)
             .catch(console.error)
-    }
+    },
+
+    deleteWorkshop: (workshopId: string) => {
+        const readWorkshops = get().readWorkshops;
```

Comment:
Why do you get the workshops here?

Reply:
i get a method from my "store" with get, so I can use it within this method.

The method does
1. get the read method
2. does the delete request
3. runs the read method (fetches the list again)